tensorflow/compiler/xla/service/elemental_ir_emitter.cc

```diff
@@ -1292,6 +1292,16 @@ StatusOr<llvm::Value*> ElementalIrEmitter::EmitComplexBinaryOp(
                     llvm_ir::EmitComparison(llvm::CmpInst::FCMP_UNE,
                                             EmitExtractImag(lhs_value),
                                             EmitExtractImag(rhs_value), b_));
+	case ComparisonDirection::kGt:
+          return Or(llvm_ir::EmitComparison(llvm::CmpInst::FCMP_OGT,
```

Comment:
Could you add a comment that we are using lexicographical comparison which is consistent with numpy behavior?